### ISSUE TITLE
does not pass to the site the first time

### ISSUE THREAD

**Issue by rnyPlanet** [thenandan/bypass-cloud-flare-ddoc-php-curl#1]
sometimes it so happens that cloudflare does not let you get to the site the first time, but it can reload the page for 5-10 minutes or more. maybe you should put while here https://github.com/thenandan/bypass-cloudflare-ddos-php-curl/blob/master/app/Main.php#L19?

**Comment by DarkneSsDz** [comment]
@rnyPlanet  hello mate , it seems that this repo doesn't work anymore because i've tried it on pastebin and it doesn't eventho the steps are correct i think that the value of the variable R isn't the one that should be passed to the request if you have any ideas please lemme know

**Comment by rnyPlanet** [comment]
> @rnyPlanet hello mate , it seems that this repo doesn't work anymore because i've tried it on pastebin and it doesn't eventho the steps are correct i think that the value of the variable R isn't the one that should be passed to the request if you have any ideas please lemme know

as I understand now everything is done in js (a lot of requests are made to the cloudflare site and new scripts are loaded), having analyzed different repositories, I realized that everyone has already stopped updating the bypass((

**Comment by DarkneSsDz** [comment]
@rnyPlanet yes i know man but i think the only thing that changed is the R value it is either getting encrypted somehow or it's not loading the R value it's getting another value that's all what i can think of right now
